pkg/experiment/metastore/index/index.go

```diff
@@ -18,17 +18,19 @@ import (
 	typesv1 "github.com/grafana/pyroscope/api/gen/proto/go/types/v1"
 	"github.com/grafana/pyroscope/pkg/experiment/block/metadata"
 	"github.com/grafana/pyroscope/pkg/experiment/metastore/index/store"
-	"github.com/grafana/pyroscope/pkg/iter"
 )
 
 const (
 	partitionDuration = 6 * time.Hour
 	// Indicates that partitions within this window are "protected" from being unloaded.
-	partitionProtectionWindow = 30 * time.Minute
+	partitionProtectionWindow = 24 * time.Hour
```

Comment:
Before we separate the index into mutable and immutable parts, it's better to minimize partition evictions